src/encode.rs

```diff
@@ -86,6 +93,11 @@ mod tests {
         assert_eq!(0.0f64.partial_cmp(&-0.0f64), Some(Ordering::Equal))
     }
 
+    #[test]
+    fn test_strings_hash_correctly() -> () {
+        assert_eq!(OrePlaintext::<u64>::from(format!("Hello world!")).0, 16562844544668089185);
```

Comment:
What is the call to `format!` for?

Reply:
Lazy coder's type conversion, from (IIRC) `&'static str` to `String`.